node/primitives/bridge/src/lib.rs

```diff
@@ -145,8 +153,56 @@ pub const TO_PANGORO_LATEST_RECEIVED_NONCE_METHOD: &str =
 
 /// Name of the `PangoroFinalityApi::best_finalized` runtime method.
 pub const BEST_FINALIZED_PANGORO_HEADER_METHOD: &str = "PangoroFinalityApi_best_finalized";
+
+/// All Polkadot-like chains allow normal extrinsics to fill block up to 75 percent.
+///
+/// This is a copy-paste from the Polkadot repo's `polkadot-runtime-common` crate.
+const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
```

Comment:
@xiaoch05 I guess we can reuse already existed const variable here https://github.com/darwinia-network/darwinia-common/blob/main/node/runtime/common/src/lib.rs#L53-L56